src/config/entities/configuration.ts

```diff
@@ -51,6 +51,9 @@ export default () => ({
     host: process.env.REDIS_HOST || 'localhost',
     port: process.env.REDIS_PORT || '6379',
   },
+  relay: {
+    limit: parseInt(process.env.RELAY_THROTTLE_LIMIT ?? `${5}`),
```

Comment:
Nit:

```suggestion
    limit: parseInt(process.env.RELAY_THROTTLE_LIMIT ?? '5'),
```

Reply:
One reason why I wrote with `${5}` was that it allows you to express numeric types using TS (one example from this file is that we can have `${5_000}` which would not work if it was a string (`5_000`).

I kept it this way also for consistency but I have no strong opinion on it and I can use a simple string as suggested 👍